tests/ts/bash-completion/consistency-check

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This file is part of util-linux.
+#
+# This file is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# Copyright (C) 2025 Christian Goeschel Ndjomouo <cgoesc2@wgu.edu>
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="consistency-check"
+
+. "$TS_TOPDIR/functions.sh"
+ts_init "$*"
+ts_skip_nonroot
+
+ts_check_test_command "$TS_TOOLS_CHECKCOMPLETION"
```

Comment:
I don't think this is necessary. The shellscript is not compiled and always available.